pep-0597.rst

```diff
@@ -21,6 +21,9 @@ This PEP proposes:
 * Add ``encoding="locale"`` option to ``TextIOWrapper``. It behaves
   like ``encoding=None`` but don't raise a warning.
 
+* Add ``io.LOCALE_ENCODING = "locale"`` constant to avoid confusing
+  ``LookupError``. unknown encoding: locale``.
```

Comment:
It seems backticks are inserted to the tail unexpectedly.

Reply:
Thank you. I fixed it in #1589.